### PR TITLE
add navbar

### PR DESCRIPTION
**Pull request by patrykmysiorski**

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.container {
+  min-height: 100vh;
+  width: 100%;
+  background-color: #485461;
```
Comment (j-stasiak) [inline src/App.module.scss:4]: do we want to write colors always as hexes or have some standarized file which will contain all the colours as variables for later usage?

**Review by j-stasiak — APPROVED** [review]

File: src/App.module.scss (hunk 1 above)
Comment (qbawalat) [reply src/App.module.scss:4]: yes we do

File: src/components/landingPage/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import styles from './LandingPage.module.scss';
+import classNames from 'classnames';
+
+interface Props {
+  isOpen: boolean;
+}
+
+const LandingPage: React.FC<Props> = ({ isOpen }) => {
+  return (
+    <div className={classNames(isOpen && styles.active)}>
```
Comment (qbawalat) [inline src/components/landingPage/LandingPage.tsx:11]: Pls use conditional classnames 
https://www.npmjs.com/package/classnames#:~:text=classNames(%27foo%27%2C%20%7B%20bar%3A%20true%20%7D)%3B%20//%20%3D%3E%20%27foo%20bar%27

File: src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import classNames from 'classnames';
+
+interface Props {
+  isOpen: boolean;
+  setIsOpen: (state: boolean) => void;
+}
+
+const Navbar: React.FC<Props> = ({ isOpen, setIsOpen }) => {
+  return (
+    <div className={classNames(styles.navbar, isOpen && styles.active)}>
```
Comment (qbawalat) [inline src/components/navbar/Navbar.tsx:12]: pls use conditional classnames

**Review by qbawalat — APPROVED** [review]

**Pull request merged**